comfino-payment-gateway.php

```diff
@@ -14,21 +14,20 @@
  * @license http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
 */
 
+use Comfino\Core;
+
 defined('ABSPATH') or exit;
 
-class ComfinoPaymentGateway
+class Comfino_Payment_Gateway
 {
-    public const VERSION = '2.3.1';
+    const VERSION = '2.4.0';
```

Comment:
jak nie public to można dodać `private`, chyba że php za stare :)

Reply:
Taa, czytam że od 7.1 a tu widać dostosowanie do 7.0

Reply:
Bo to jest bład. Jak się okazało trzymamy jednak zgodność z PHP 7.0.

Reply:
Poprawiłem readme.txt. W nagłówku modułu PHPDoc było już 7.0 wpisane.